convert2rhel/subscription.py

```diff
@@ -469,7 +431,12 @@ def __call__(self):
 
             try:
                 if self.password:
-                    loggerinst.info("Registering via username/password: %s" % " ".join(utils.hide_secrets(self.args)))
+                    if self.org:
+                        loggerinst.info("Organization: %s", self.org)
+                    loggerinst.info("Username: %s", self.username)
```

Comment:
Please replace these two with asterisks based on https://github.com/oamg/convert2rhel/pull/725.

You can perhaps define this `obfuscation_string` as a constant on a module level and then use it in this `subscription.py` module as well as in `utils.py`.
https://github.com/oamg/convert2rhel/blob/v1.1/convert2rhel/utils.py#L722

Reply:
Or if this is ready to merge, we can do so and I can update #725.

Reply:
@bocekm it is ready to merge, it would be great if you could update #725 thanks!

Reply:
Done.